tests/jest/BookNavigator/downloads/downloads-provider.test.js

```diff
@@ -29,39 +29,39 @@ afterEach(() => {
 });
 
 describe('Downloads Provider', () => {
-  it('constructor - initial setup', () => {
+  test('constructor - initial setup', () => {
     const isBookProtected = false;
     const provider = new DownloadsProvider(isBookProtected);
 
-    expect(provider.id).to.equal('downloads');
-    expect(provider.icon).to.exist;
-    expect(fixtureSync(provider.icon).tagName).to.equal('IA-ICON-DL');
-    expect(provider.label).to.equal('Downloadable files');
-    expect(provider.menuDetails).to.exist;
-    expect(provider.component).to.exist;
+    expect(provider.id).toEqual('downloads');
+    expect(provider.icon).toBeDefined();
+    expect(fixtureSync(provider.icon).tagName).toEqual('IA-ICON-DL');
+    expect(provider.label).toEqual('Downloadable files');
+    expect(provider.menuDetails).toBeDefined();
+    expect(provider.component).toBeDefined();
 
     provider.update(downloadableTypes);
 
-    expect(provider.isBookProtected).to.equal(false);
+    expect(provider.isBookProtected).toEqual(false);
 
-    expect(provider.downloads[0].type).to.equals("PDF");
-    expect(provider.downloads[1].type).to.equals("ePub");
+    expect(provider.downloads[0].type).toEqual("PDF");  //.to.equals( )
+    expect(provider.downloads[1].type).toEqual("ePub"); //.to.equals( )
```

Comment:
Don't need these comments here

```suggestion
    expect(provider.downloads[0].type).toEqual("PDF");
    expect(provider.downloads[1].type).toEqual("ePub");
```